hybrasyl/Connection.cs

```diff
@@ -77,6 +80,48 @@ public static void DeregisterClient(Client client)
                 }
             }
         }
+
+        public static byte[] RequestEncryptionKey(string endpoint, IPAddress remoteAddress)
+        {
+            byte[] key;
+
+            var webReq = WebRequest.Create(new Uri(endpoint));
+            webReq.ContentType = "application/json";
+            webReq.Method = "GET";
+
+            var response = webReq.GetResponse();

```

Comment:
no error handling?

Reply:
not needed at this point. need to test further to see what possible errors can come back, but the response should always return

Reply:
🔢